src/coreComponents/mesh/generators/CellBlockManager.hpp

```diff
@@ -0,0 +1,236 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file CellBlockManager.hpp
+ */
+
+#ifndef GEOSX_MESH_CELLBLOCKMANAGER_H_
+#define GEOSX_MESH_CELLBLOCKMANAGER_H_
+
+#include "mesh/generators/CellBlockManagerABC.hpp"
+#include "mesh/generators/CellBlock.hpp"
+
+namespace geosx
+{
+
+/**
+ * @class CellBlockManager
+ * @brief The CellBlockManager class provides an interface to ObjectManagerBase in order to manage CellBlock data.
+ */
+class CellBlockManager : public CellBlockManagerABC
```

Comment:
In the next PR when we will have a `CellBlockManager` dedicated to arbitrary polyhedral grids, we will have to define another name for that :)

Reply:
Fatalitas!